Cue/app/actions/library.js

```diff
@@ -110,7 +110,15 @@ async function syncDeck(change) : PromiseAction {
       }
     }
   } else if (change.action === "edit") {
-    serverDeck = await LibraryApi.editDeck(change)
+    if (change.name || change.tags || change.public
+      || (change.cards && change.cards.find(card => card.front || card.back || card.poistion))) {
```

Comment:
Right now this will cause the server a 400 error.

2 ways to fix that.
1) we can either send 2 requests from the client, 1 for every all the card edits and 1 for all the flags.
2) server supports all action propertys as optional, so that in 1 deck edit request we can edit 1 card and just flag another